.github/workflows/ci.yml

```diff
@@ -193,6 +193,24 @@ jobs:
       package-dist-directory: ./apps/pie-storybook/dist
       bucket-name-preview: 'pie-storybook-preview'
       bucket-name-main: 'pie-storybook'
+      sub-domain-suffix: 'storybook'
+    secrets: inherit
+
+  deploy-storybook-testing:
```

Comment:
Would be great to comment on some of the context of this inline. So even just putting this would be useful:

`# Workflow task that deploys an instance of storybook used for test suites to run against`